popup/popup.js

```diff
@@ -21,6 +21,16 @@ $("#options_link").click(function() {
   window.close();
 });
 
+// Handle mouseover of all infobuttons
```

Comment:
With the suggestion above regarding the language in the popup, we'll no longer need to handle infobuttons, so this can be removed.

Reply:
Okay, I'll pull that out!